src/gui/screen_menu_esp_update.cpp

```diff
@@ -0,0 +1,332 @@
+#include "sys.h"
+#include "gui.hpp"
+#include <stdbool.h>
+#include "RAII.hpp"
+#include "screen_menu.hpp"
+#include "screen_menus.hpp"
+#include "WindowMenuItems.hpp"
+#include "i18n.h"
+#include "ScreenHandler.hpp"
+#include "main.h"
+#include "dbg.h"
+#include "string.h"
+
+#include "cmsis_os.h"
+#include "stm32f4xx_hal.h"
+#include "stm32_port.h"
+#include "esp_loader.h"
+// #include "example_common.h"
+#include "esp_flasher.h"
+#define HIGHER_BAUDRATE 115520
```

Comment:
Isn't this a typo? The standard baudrate is 115200

Reply:
It's not used any more. It will be deleted.